File: intro-prog/lista 2/ex11.py
"""
11. Escreva um programa que lê um caractere (string de 1 char) e informa se o mesmo é uma letra
maiúscula.
a. Considere como 'letra', todos os caracteres da língua portuguesa na tabela Unicode:
https://usefulwebtool.com/characters-portuguese
"""

char = int(input("Digite um numero "))

if char >= 65 and char <= 90:
    print("É a letra %c" % char)
else:
    print("Não é uma letra")

File: Algoritmos/Lista 1/ex11.c
/*11. Escreva um programa que lê um caractere da entrada. Se não for letra, informa. Se for letra,
verifica se é maiúscula (e passa para maiúscula se necessário). Ao final, deve informar a letra
digitada em maiúsculo.*/

#include <stdio.h>

int main(){
    short carac;

    printf("Digite o char: ");
    scanf("%hd", &carac);

    if(carac >= 65 && carac <= 90){
        printf("Letra Maíscula!\n");
    }
    else if(carac >= 97 && carac <= 122){
        printf("Letra Minúscula!\n");
    }
    else{
        printf("Não é uma letra!\n");
    }
}